### PR TITLE
[Site Name] Adjusts design picker buttons for treatment variation

### PR DESCRIPTION
**Pull request by antonis**
## Description
This PR adjusts the design picker `Choose` and `Skip` buttons to communicate to the user that their action will create the site.

|Design Chooser(Control)|Design Chooser(Treatment)|
|---|---|
|Screenshot_20220419_115412|Screenshot_20220426_080559|

|Design Preview(Control)|Design Preview(Treatment)|
|---|---|
|Screenshot_20220419_115424|Screenshot_20220419_114941|

### Treatment variation
<details>
<summary>1. Toggle ON the Site Name feature</summary>

1. From My Site Go to Me → App Settings → Debug settings
2. Scroll to Features in development section and tap on SiteNameFeatureConfig
3. Tap RESTART THE APP button
4. Since this feature is A/B tested go to **Abacus**, find the `wpandroid_site_name_v1` experiment and assign to your test a8c-user the treatment variation

**Alternatively hardcode the feature on by returning `true` [here](https://github.com/wordpress-mobile/WordPress-Android/blob/521e6b3a296339bec026f8ee2648eab326d31129/WordPress/src/main/java/org/wordpress/android/util/config/SiteNameFeatureConfig.kt#L19)** 
</details>

2. Start the Site Creation flow
3. Choose an intent for your site or skip
4. Enter a site name or skip
5. Select a design
6. Verify that the `skip` and `choose` button are changed to `skip and create` and `create site` respectively
7. Preview a design and verify that the `choose` button is changed to `create site`

### Control variation
1. Toggle OFF the Site Name feature
2. On steps 6 and 7 above verify that the buttons are not changed

## Regression Notes
1. Potential unintended areas of impact
N/A

3. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

4. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by antonis** [comment]
The skip button change was reverted with 294c89b157f8efd5a21c2de9534806e60b05857d

**Comment by AliSoftware** [comment]
FYI: Moving this to milestone 19.8 since I'm doing 19.7 code freeze and this is PR still draft anyway 🙂

**Review by ovitrif — APPROVED** [review]
Thank you for tackling this @antonis 🙇 

Code LGTM, and I tested both cases: with the site name experiment on and off, and it worked as expected :shipit:

~We'd have to change the target branch if we want this to reach **19.7**, since it was cut already and we have a release branch.~
NVM that was tackled already and we're targeting the right branch to have it land in `19.7` 🚀

File: WordPress/src/main/res/values/strings.xml
Comment (AliSoftware) [inline WordPress/src/main/res/values/strings.xml:3342]: @antonis Unfortunately, since that PR was merged into `release/19.7` and thus _after_ the code freeze, it wasn't included in the frozen strings sent to our Translation Vendor last Tuesday for this version.

Since I caught this up today by chance while working on the release branch, and I've decided to update the frozen strings file to include that new one (via: d55cc88), so that at least it gets imported into GlotPress ASAP and hopefully translated in the Mag16 by the community in time, even if our translator vendor (who usually ensure Mag16 coverage) didn't get that one in time. 🤞 

For the record and for next time, please be sure to ping your Release Manager when you need to update string copies on PRs done post-code-freeze and targeting the `release/*` branch, or when you decide to change the target branch of a PR from `trunk` to `release/*`   🙇

**Comment by AliSoftware** [comment]
> ~We'd have to change the target branch if we want this to reach 19.7, since it was cut already and we have a release branch.~
> NVM that was tackled already and we're targeting the right branch to have it land in 19.7 🚀

Well tbh it would have been useful — especially since I initially communicated that the PR would not make the cut when I did the code freeze — to ping me to mention that you wanted this PR to make it to 19.7 anyway when you decided to restore milestone from 19.8 to 19.7 and re-target `release/*`. 🥺 

That would have allowed me to spot and warn you earlier about the fact that the new string would not have been included in the already-frozen ones, allowing us to decide if it was still worth it to make this one land to `release/19.7` or not, or taking appropriate measures to have it included in translations requests before (e.g. if I saw this 2 days ago — when this was merged —, we would still have had time to make this new string be included in the ones sent to our Translation Vendor, instead of it missing the deadline) 😕

**Comment by antonis** [comment]
Thank you for your efforts and sorry for the miscommunication @AliSoftware 😞 
We will make sure that any translation change from now on will be included before the code freeze.

**Comment by AliSoftware** [comment]
This has now been shipped as part of `19.7-rc-2` 🎉

**Comment by ovitrif** [comment]
> Well tbh it would have been useful — especially since I initially communicated that the PR would not make the cut when I did the code freeze — to ping me to mention that you wanted this PR to make it to 19.7 anyway when you decided to restore milestone from 19.8 to 19.7 and re-target release/*. 🥺

> That would have allowed me to spot and warn you earlier about the fact that the new string would not have been included in the already-frozen ones, allowing us to decide if it was still worth it to make this one land to release/19.7 or not, or taking appropriate measures to have it included in translations requests before (e.g. if I saw this 2 days ago — when this was merged —, we would still have had time to make this new string be included in the ones sent to our Translation Vendor, instead of it missing the deadline) 😕

Thank you @AliSoftware and sorry for not pinging you. We're learning a bit more about the translations process and how it relates to release management. Definitely going to ping you and not merge before you're up-to-date if this situation is ever needed again. Ideally, as Antonis said, we'll do more effort to update the translations before the code-freeze, and target any string updates for the next release if needed.

**Pull request merged**
